eng/pipelines/templates/scripts/mashup-doc-index.yml

```diff
@@ -0,0 +1,85 @@
+parameters: 
+  TargetFolder: ''
+
+steps:
+  - task: PythonScript@0
+    displayName: MashUp Generated Index Site so its served from default site location
+    inputs:
+      scriptSource: inline
+      script: |
+        import argparse
+        import os
+        import logging
+        import re
+        import shutil
+        from io import open
+
+        SITE_INDEX = r'${{ parameters.SourceDirectory }}\docfx_project\_site'
+        TOC_HTML_REGEX = r"\.\./toc.html"
+        NAV_TOC_HTML_REGEX = r"api/"
+        PREV_DIR_REGEX = r"\.\./"
+
+        def locate_htmlfiles(directory):
+            html_set = []
+            for root, dirs, files in os.walk(directory):
+                for file in files:
+                    html_set.append(os.path.join(root, file))
+            return html_set
+    
+        def process_html(content):
+            content = re.sub(TOC_HTML_REGEX, 'navtoc.html', content)
+            content = re.sub(PREV_DIR_REGEX, '', content)
+            return content
+    
+        def process_navtoc(content):
+            content = re.sub(NAV_TOC_HTML_REGEX, '', content)
+            return content
+
+        if __name__ == "__main__":
+            html_files = locate_htmlfiles(os.path.join(SITE_INDEX, 'api'))
+
+            print(SITE_INDEX)
+            navtoc_location = os.path.join(SITE_INDEX, 'toc.html')
+
+            print(navtoc_location)
```

Comment:
Mix between `print` and `logging`. It's not a big deal, but try to choose one and keep it consistent. (logging is definitely preferred)

Reply:
Ill clean this up